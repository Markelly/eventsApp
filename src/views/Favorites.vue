<template>
  <div class="favorites">
    <h1>FAVORITES</h1>
    <EventsList/>
  </div>
</template>

<script>
import EventsList from "../components/EventsList";
export default {
  name: "Favorites",
  components: { EventsList }
};
</script>

<style scoped lang="scss">
@import "../styles/events-app";
</style>
