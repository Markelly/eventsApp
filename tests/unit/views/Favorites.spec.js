import Favorites from "../../../src/views/Favorites";
import { createLocalVue, shallowMount } from "@vue/test-utils";

describe("Favorites", () => {
  const localVue = createLocalVue();
  let wrapper;

  test("should render the component", () => {
    wrapper = shallowMount(Favorites, { localVue });
    expect(wrapper).toMatchSnapshot();
  });
});
