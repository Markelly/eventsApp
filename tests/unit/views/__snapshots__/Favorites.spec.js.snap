// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Favorites/> should render the component 1`] = `
<page-stub pagetitle="FAVORITES" buttonlabel="See all events">
  <div>
    <eventslist-stub></eventslist-stub>
  </div>
</page-stub>
`;
